python/ray/_private/worker.py

```diff
@@ -1263,6 +1263,15 @@ def init(
         usage_lib.record_library_usage("client")
         return ctx
 
+    if kwargs.get("allow_multiple"):
+        raise RuntimeError(
+            "`allow_multiple` argument is passed to `ray.init` when the "
+            "ray client is not used ("
+            "https://docs.ray.io/en/master/cluster/running-applications/job-submission"
```

Comment:
Feel we need to find a way to link check those links as well.

Reply:
how do we do this now?

Reply:
Followed up from the slack. Let's do this in a separate PR?